### PR TITLE
chore: add missing models (3.5 sonnet and 3 opus)

### PR DESCRIPTION
**Pull request by tmokmss**
Added Opus 3 and Sonnet 3.5 to [BedrockFoundationModel](https://awslabs.github.io/generative-ai-cdk-constructs/apidocs/namespaces/bedrock/classes/BedrockFoundationModel.html).

Note that both models are supported on Agents for Bedrock: https://docs.aws.amazon.com/bedrock/latest/userguide/agents-supported.html

---
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the project license.

### CODE REVIEW

File: src/cdk-lib/bedrock/models.ts [hunk 1]
```diff
@@ -57,6 +57,14 @@ export class BedrockFoundationModel {
     'amazon.titan-text-express-v1',
     { supportsAgents: true },
   );
+  public static readonly ANTHROPIC_CLAUDE_3_5_SONNET_V1_0 = new BedrockFoundationModel(
+    'anthropic.claude-3-5-sonnet-20240620-v1:0',
+    { supportsAgents: true },
+  );
+  public static readonly ANTHROPIC_CLAUDE_OPUS_V1_0 = new BedrockFoundationModel(
```
Comment (tmokmss) [inline src/cdk-lib/bedrock/models.ts:64]: Maybe we should deprecate names like `ANTHROPIC_CLAUDE_SONNET_V1_0` and rename it to `ANTHROPIC_CLAUDE_3_SONNET_V1_0` for consistency.

**Comment by krokoko** [comment]
@tmokmss build is failing due to some files not pushed, could you please run a full build locally and push the missing files ? (projen build). Thanks !

**Comment by tmokmss** [comment]
@krokoko Fixed it, thanks!

File: src/cdk-lib/bedrock/models.ts (hunk 1 above)
Comment (laithalsaadoon) [reply src/cdk-lib/bedrock/models.ts:66]: Agree, although we should probably have a separate PR

**Review by laithalsaadoon — APPROVED** [review]
LGTM

**Review by krokoko — APPROVED** [review]

**Pull request merged**